src/main/java/net/minecraftforge/client/extensions/common/IClientMobEffectExtensions.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.extensions.common;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.gui.Gui;
+import net.minecraft.client.gui.screens.inventory.EffectRenderingInventoryScreen;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.effect.MobEffectInstance;
+import net.minecraftforge.fml.LogicalSide;
+
+import java.util.function.Consumer;
+
+/**
+ * {@link LogicalSide#CLIENT Client-only} extensions to {@link MobEffect}.
```

Comment:
The label should be plain text, so the tag should be `{@linkplain}`.
```suggestion
 * {@linkplain LogicalSide#CLIENT Client-only} extensions to {@link MobEffect}.
```